java/src/jmri/jmrit/logixng/actions/ActionLocalVariable.java

```diff
@@ -33,9 +33,9 @@
     private String _otherLocalVariable = "";
     private String _formula = "";
     private ExpressionNode _expressionNode;
-    private boolean _listenToMemory = true;
-    private boolean _listenToBlock = true;
-    private boolean _listenToReporter = true;
+    private boolean _listenToMemory = false;
+    private boolean _listenToBlock = false;
+    private boolean _listenToReporter = false;
```

Comment:
I suggest a note in the release notes about this change. That the default for `listen` has changed from `on` to `off`.

Reply:
The default was actually false.  When the local variable is added, the swing code defaults to false with the checkbox unchecked.

This just makes the default states consistent.

Reply:
Thanks.